test/files/run/repl-serialization.scala

```diff
@@ -14,6 +14,9 @@ object Test {
   def run(): Unit = {
     val settings = new Settings()
     settings.Yreplclassbased.value = true
+    // The postSetHook of the above implies settings.YreplMagicImport.value = true
```

Comment:
no longer tue - do we need `settings.YreplMagicImport.value = true` now?

Reply:
No, I think not.  I think the point is that Jason's "REPL dead backreference elimination" works with and without.  So it should be tested under `settings.Yreplclassbased.value = true` (what Spark uses) and the default `YreplMagicImport`.  I'll drop the comment.